etcdutl/snapshot/util.go

```diff
@@ -20,12 +20,23 @@ import (
 
 type revision struct {
 	main int64
+	sep  byte
 	sub  int64
 }
 
+// The binary representation of the revision is 17 bytes:
+//
+// [8 bytes: main] [1 byte: sep] [8 bytes: sub]
 func bytesToRev(bytes []byte) revision {
```

Comment:
Don't like the idea of copying this code. It's too fragile. Still making this public might not be best solution. Can you  comment that `bytesToRev` and `revToBytes` functions should be synced with function server.